app/views/partials/daAlert.scala.html

```diff
@@ -0,0 +1,12 @@
+@import java.util.UUID
+@(heading: String, content: Html)
+<div class="da-alert da-alert--default">
```

Comment:
Is this an existing component in the design system?

Reply:
I copied it of the prototype. I had a look at the govuk design system but there doesn't seem to be a component like this. Added as a partial as it is used in metadata results error and non-error report pages